backend/assignments/1/src/main/java/jav/assignment/Main.java

```diff
@@ -0,0 +1,47 @@
+package jav.assignment;
+
+import jav.assignment.globalObjects.GlobalData;
+import jav.assignment.readFiles.ReadJSON;
+import com.fasterxml.jackson.databind.JsonNode;
+
+import java.io.IOException;
+import java.util.concurrent.CountDownLatch;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        try {
```

Comment:
More modularity expected.
Move utils, core logic etc in seperate classes/packages